src/common/mixin/store-behaviour.js

```diff
@@ -42,39 +42,41 @@ const storeMixin = {
 
     /**
      * Get the state informations from the store.
-     * @param  {array} filterNodes - An object containing nodes key to update.
+     * @param  {array} filterNodesArg - An object containing nodes key to update.
      * @returns {object} - The js object constructed from store data.
      */
-    _getStateFromStores(filterNodes = []) {
+    _getStateFromStores(filterNodesArg) {
         if (this.getStateFromStore) {
             return this.getStateFromStore();
```

Comment:
I wonder if we shouldn't push `filterNodesArg` to the override method `getStateFromStore`